packages/web-components/src/components/cbp-structured-list-item/cbp-structured-list-item.scss

```diff
@@ -1,11 +1,31 @@
+/** todo: update doc
+ * @prop 
+*/
+:root {
+  --cbp-structured-list-item-padding: var(--cbp-space-4x);
+
+  --cbp-structured-list-item-color-background-danger: var(--cbp-color-danger-lighter);
```

Comment:
The color variants shouldn't be up here. They should be modifying the base variable in their selector.

Reply:
discussed on call, marking as resolved based on that discussion